### ISSUE TITLE
subtract, merge, intersect methods should work with bemfiles

### ISSUE THREAD

**Issue by zxqfox** [bem-sdk/bem-decl#48]
BemFile is a structure that has entity and tech.
Normalize v2 returns items as bemfiles.
`bem-graph` also uses this format to operate with deps.

Seems like we should support it everywhere.

**Comment by zxqfox** [comment]
Decl algebra:
```js
// U — merge
[{ entity: A }] U [{ entity: A, tech: '1' }] → [{ entity: A }]
[{ entity: A, tech: '1' }] U [{ entity: A, tech: '2' }] →
    → [{ entity: A, tech: '1' }, { entity: A, tech: '2' }]

// / — subtract
[{ entity: A }] / [{ entity: A, tech: 'specified' }] → [{ entity: A }]
[{ entity: A, tech: 'specified' }] / [{ entity: A }] → []

// П — intersect
[{ entity: A }] П [{ entity: A, tech: 'specified' }] → [{ entity: A, tech: 'specified' }]
[{ entity: A, tech: '1' }] П [{ entity: A, tech: '2' }] → []
```

**Comment by blond** [comment]
closed favor of #81
